dsgrid/utils/dataset.py

```diff
@@ -16,21 +16,19 @@
 @track_timing(timer_stats_collector)
 def map_and_reduce_stacked_dimension(df, records, column):
     if "fraction" not in df.columns:
-        df = df.withColumn("fraction", F.lit(1))
+        df = df.withColumn("fraction", F.lit(1.0))
```

Comment:
I changed this because it felt weird to add an integer column that should be a float. It was working correctly, but I think this reads better.